NBitcoin/Secp256k1/ECPubKey.cs

```diff
@@ -304,6 +305,29 @@ private bool secp256k1_eckey_pubkey_tweak_add(ECMultContext ctx, ref GE key, in
 			return true;
 		}
 
+		/// <summary>
+		/// The original function name is `secp256k1_ec_pubkey_combine`
+		/// </summary>
+		public static bool TryCombine(Context ctx, IEnumerable<ECPubKey> pubkeys, out ECPubKey? combinedPubKey)
+		{
+			if (pubkeys == null)
+				throw new ArgumentNullException(nameof(pubkeys));
+
+			var count = pubkeys.Count();
+			if (count == 0)
+			{
+				combinedPubKey = null;
+				return false;
+			}
+			var ptj = GEJ.Infinity;
+			foreach (var p in pubkeys)
+			{
+				ptj = ptj.Add(in p.Q);
+			}
+			combinedPubKey = new ECPubKey(ptj.ToGroupElement(), ctx);
```

Comment:
Should be jacobian.
` secp256k1_ge_set_gej(&Q, &Qj);`

Reply:
I don't understand, I thought what these two lines do is
```c
secp256k1_ge_set_gej(&Q, &Qj);
secp256k1_pubkey_save(pubnonce, &Q);
```
1. Convert jacobian coordinate (Qj) into non jacobian-coordinate representation Q and
2. initialize pubkey (pubnonce) with that Q.
 
so it seems current code is fine.